test/testdrive/system-cluster.td

```diff
@@ -56,3 +56,96 @@ INSERT INTO temp SELECT * FROM temp
 2
 1
 2
+
+> SET CLUSTER TO mz_introspection
+
+$ set-regex match=(\(\d+\s=\s|u\d+) replacement=
+
+? EXPLAIN SHOW DATABASES
```

Comment:
Can this be moved to a .slt file because it supports rewrites? EXPLAIN plans change semi-regularly, and these will have to be modified by hand.

Reply:
Moved